vector/src/debug/java/im/vector/app/features/debug/features/DebugVectorFeatures.kt

```diff
@@ -106,4 +108,5 @@ object DebugFeatureKeys {
     val onboardingSplashCarousel = booleanPreferencesKey("onboarding-splash-carousel")
     val onboardingUseCase = booleanPreferencesKey("onbboarding-splash-carousel")
     val onboardingPersonalize = booleanPreferencesKey("onbboarding-personalize")
+    val forceLoginFallback = booleanPreferencesKey("force-login-fallback")
```

Comment:
Maybe better to add this simple boolean to the screen `DebugPrivateSettingsFragment` as suggested by @ouchadam , since this is really something for internal test and not a feature we can disable or enable.

Reply:
that was my thinking ^^^ I don't see us wanting to promoting this setting for production/all users